### PR TITLE
Add List css

### PR DESCRIPTION
**Pull request by rahulcs754**
Added List CSS 
1. normal list 
2. stacked list 
3. nested list 
4. checkbox list 

image

image

### CODE REVIEW

File: css/components/list.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+.list-border{
+    border: 1px solid var(--light-black);
+}
+
+.list-title{
+    font-size: 1.3rem;
+    padding: 1rem;
+    font-weight: 700;
+    border-bottom: 1px solid #d9d9d9;
```
Comment (tanishq20) [inline css/components/list.css]: add color in `var`

**Review by tanishq20 — COMMENTED** [review]
Everything looks good. Just one small change.

**Review by shubhamkumar648 — APPROVED** [review]
code looks for me

**Review by tanishq20 — APPROVED** [review]

File: css/components/list.css (hunk 1 above)
Comment (rahulcs754) [reply css/components/list.css]: done

**Pull request merged**